### DISCUSSION TITLE
mob drop

### DISCUSSION THREAD

**Discussion by jigglypuff2731** [NCBPFluffyBear/SlimeCustomizer#14] [General]
so i made a custom mob spawn recipe and it drops a key item how in the guide do i show the mob must be killed not used as a recipe

**Comment by NCBPFluffyBear** [comment]
It should say at the side that it needs to be killed, doesn't it?

**Reply by jigglypuff2731** [reply]
no i want to know how to do that i did the drop part with nbt but i dont know how to show it as a drop

**Reply by jigglypuff2731** [reply]
![Screen Shot 2021-07-25 at 3 36 11 PM](https://user-images.githubusercontent.com/80793133/126915985-fbd19a19-cda0-4966-9b80-8bd747654d52.png)
![Screen Shot 2021-07-25 at 3 36 26 PM](https://user-images.githubusercontent.com/80793133/126915987-1ce7c360-03b7-46a4-b5dc-017a97508122.png)
![Screen Shot 2021-07-25 at 3 36 37 PM](https://user-images.githubusercontent.com/80793133/126915988-dffee6e1-632e-4084-abec-eb357a99d64c.png)
but i want it to be
![Screen Shot 2021-07-25 at 3 39 51 PM](https://user-images.githubusercontent.com/80793133/126915996-1c6d29bc-7c9c-46eb-bcc4-057d5bca462d.png)

**Reply by jigglypuff2731** [reply]
the mob drop is from the circuit board recipe

**Reply by NCBPFluffyBear** [reply]
There is no way to do that atm, I'll keep it in mind, might add in future updates
